### PR TITLE
Document more NavigationMesh properties

### PR DESCRIPTION
**Pull request by skyace65**
Adds a description for Agent Height, Agent Radius and Cell Height.

### CODE REVIEW

File: doc/classes/NavigationMesh.xml [hunk 1]
```diff
@@ -87,8 +88,10 @@
 			The maximum angle a slope can be at for navigation to be generated on it.
 		</member>
 		<member name="agent/radius" type="float" setter="set_agent_radius" getter="get_agent_radius" default="0.6">
+			Determines where the edge of a navigation mesh is so an agent will not overlap with another mesh, or stand over nothing.
```
Comment (Calinou) [inline doc/classes/NavigationMesh.xml:91]:
```suggestion
			Determines where the edge of a navigation mesh is. This way, an agent will not overlap with another mesh or stand over nothing.
```

**Review by Calinou — DISMISSED** [review]
Should be good to merge after applying the suggestion.

**Comment by skyace65** [comment]
Fixed

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**